src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/09-services.e2e-spec.ts

```diff
@@ -24,4 +24,16 @@ describe('Services page', () => {
 
     services.deleteService('mds.test');
   });
+
+  it('should create snmp-gateway service with version V2c', () => {
+    services.navigateTo('create');
+    services.addService('snmp-gateway', false, 'V2c');
+    services.checkExist('snmp-gateway', true);
```

Comment:
once the snmp-gateway is deployed its daemons will be in a running state right? Can we check that too? If the daemon is running or not?
```suggestion
    services.clickServiceTab('snmp-gateway', 'Details');
    cy.get('cd-service-details').within(() => {
      services.checkServiceStatus('snmp');
    });
```

Reply:
Sure